client/src/grade/Grade.js

```diff
@@ -8,75 +9,143 @@ class Grade extends React.Component{
+                    question_id: 1,
+                    question_text: "What is the one thing that all wise men, regardless of their religion or politics, agree is between heaven and earth?",
+                    response_id: 1,
+                    response_value: "The US Senate passed a massive, $2 trillion stimulus package late Wednesday night"
+                    },
+                    {
+                    eval_id: 2,
+                    prompt_text: "Dont judge a book by the cover People take a look at the world and discover That beauty is the word that I think of when I see the different colors of skin And Ill rejoice and sing for them",
+                    question_id: 2,
+                    question_text: "What is the one thing that all wise men, regardless of their religion or politics, agree is between heaven and earth?",
+                    response_id: 1,
+                    response_value: "The US Senate passed a massive, $2 trillion stimulus package late Wednesday night"
+                    },
+                    {
+                    eval_id: 8,
+                    prompt_text: "Good vibes Were bringing only good vibes People walking around talking down on others You cant know yourself without knowing about the other",
+                    question_id: 4,
+                    question_text: "There is this one man who killed his mother. He was born before his father, and married over 100 women without divorcing any one. Yet, he was considered normal by all of his acquaintances. Why?",
+                    response_id: 8,
+                    response_value: "In New York, the epicenter of the COVID-19 outbreak in the U.S., Gov"
+                    }],
+            loading: true
         }
        
-        this.handleFormChange = this.handleFormChange.bind(this);
         this.handleSubmit = this.handleSubmit.bind(this);
         this.handleSliderChange = this.handleSliderChange.bind(this);
-
+        this.makeEvals = this.makeEvals.bind(this);
+        this.handleAddE = this.handleAddE.bind(this);
+        this.filler = this.filler.bind(this);
+        window.onload = this.makeEvals
     }
+
     handleSliderChange (event) {
         this.setState({value: event.target.value});
     }
 
-    handleFormChange(event) {
-        this.setState({ [event.target.name]: event.target.value });
+    makeEvals(event){
+        var x = this.state.evalInfo.length
+        for(var i = 0; i < x; i++){
+            this.handleAddE()
+        }
+        this.state.loading = false
+    }
+
+    handleAddE(event){
+        let e = this.state.evaluation
+        let esz = this.state.esz
+        e[this.state.esz] = {evaluationID:  "", grade: 0}
+        this.setState({esz: ++esz, evaluation: e})
+    }
+
+    handleFormChange(q_ID, e_ID, event) {
```

Comment:
When I try to modify the value on the bar I get this error:
`Cannot set property 'evaluationID' of undefined`
Don't use `esz` like I did, you only need to make an Id for each evaluation. You do this by providing a unique `name` parameter to the bar, then here you put `this.setState({[event.target.name] : +event.target.value});`